docs/BigQueryTable-batchsource.md

```diff
@@ -39,6 +39,12 @@ Datasets are top-level containers that are used to organize and control access t
 Each record is composed of columns (also called fields).
 Every table is defined by a schema that describes the column names, data types, and other information.
 
+**Partition From Date**: It's inclusive partition start date. It should be a String with format "yyyy-MM-dd". 
```

Comment:
Inclusive partition start date, specified as 'yyyy-MM-dd'. For example, '2019-01-01'. If no value is given, all partitions up to the partition end date will be read.

Reply:
In the descriptions we call them 'start' and 'end' vs 'from' and 'to'. I think that is more natural, so let's change the labels as well.

Reply:
Done.